packages/network/lib/cors.ts

```diff
@@ -71,26 +74,75 @@ export function getDomainNameFromParsedHost (parsedHost: ParsedHost) {
   return _.compact([parsedHost.domain, parsedHost.tld]).join('.')
 }
 
-export function urlMatchesOriginPolicyProps (urlStr, props) {
-  // take a shortcut here in the case
-  // where remoteHostAndPort is null
+export function urlMatchesSuperDomainOriginProps (urlStr, props) {
+  return urlMatchesSameSiteProps(urlStr, props, true)
```

Comment:
Is this really the same as same-site? I thought with same-site the port or protocol didn't matter, but it should matter for superDomainOrigin! (this name is funny)

Full discosure: I had to look up the difference between same-site and same-origin today: https://web.dev/same-site-same-origin/

Reply:
ah, i see you're forcing the ports to be strict

Reply:
> Is this really the same as same-site

With `strictPortMatch=false` it is. Otherwise it is just a `superDomainOrigin` match. I brought this up in a thread with @mschile [here](https://github.com/cypress-io/cypress/pull/23879#discussion_r976778956) as I had no idea what to actually call this function. I added some JSDocs for `strictPortMatch` to suggest that if true, this is really a superDomainOrigin match and not a samesite match (I think I should explicitly say that)

Reply:
> Full discosure: I had to look up the difference between same-site and same-origin today: https://web.dev/same-site-same-origin/

I understand 😆 . I put together a big cross reference chart in the feature branch #23872 to distinguish all the options. It's confusing

Reply:
How about flexing on the policy to match:

```js
export function urlMatchesPolicy ({ url, props, policy }: {url: string, props?: ParsedHostWithProtocolAndHost, policy: 'same-site'| 'same-super-domain-origin'})
```

Reply:
I like it. much more readable. Might be able to include the `same-origin` check in here as well